src/CloudApi/ClientService.php

```diff
@@ -42,7 +42,6 @@ public function getClient(): Client {
     $user_agent = sprintf("acli/%s", $this->application->getVersion());
     $client->addOption('headers', [
       'User-Agent' => [$user_agent],
-      'Accept'     => 'application/json',
```

Comment:
This forces all API calls (even internal ones) to return JSON. That's obviously bad if what we want from the API is a DB backup as an octet stream. So I moved this to only be set on public API calls.